dali/pipeline/proto/dali.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2017-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I think it's rather `2017, 2021`

Reply:
If anything it should be 2017-2018, 2021, but my script doesn't handle that.

Reply:
Okay, let's not complicate it. 👍